### PR TITLE
Adjust autopayScheduledTime attribute

### PR DESCRIPTION
**Pull request by vova07**

### CODE REVIEW

File: spec/components/schemas/Invoices/Invoice.yaml [hunk 1]
```diff
@@ -92,7 +92,7 @@ properties:
         - $ref: "#/components/schemas/InvoiceDiscount"
   autopayScheduledTime:
     description: Invoice autopay scheduled time
-    readOnly: true
+    readOnly: false
```
Comment (ashkarpetin) [inline spec/components/schemas/Invoices/Invoice.yaml]:
```suggestion
```

Comment (vova07) [reply spec/components/schemas/Invoices/Invoice.yaml]: @ashkarpetin I need to make it explicitly `false` to override the value from `ServerTimestamp`

Comment (ashkarpetin) [reply spec/components/schemas/Invoices/Invoice.yaml]: Then you are using wrong schema:
```
type: string
description: Read-only timestamp, automatically assigned on back-end.
format: date-time
readOnly: true
```

File: spec/components/schemas/Invoices/Invoice.yaml [hunk 2]
```diff
@@ -92,7 +92,7 @@ properties:
...
     allOf:
```
Comment (ashkarpetin) [inline spec/components/schemas/Invoices/Invoice.yaml]:
```suggestion
```

File: spec/components/schemas/Invoices/Invoice.yaml (hunk 1 above)
Comment (ashkarpetin) [reply spec/components/schemas/Invoices/Invoice.yaml]: I suggest add this and remove ` $ref: "#/components/schemas/ServerTimestamp"`

Comment (slavcodev) [reply spec/components/schemas/Invoices/Invoice.yaml]: Yes, the purpose of `ServerTimestamp` schema is the read-only time which is set by server. For writable fields, no reasons to use this schema.

**Pull request merged**